pkg/trace/config/apply_test.go

```diff
@@ -59,3 +63,59 @@ func TestSplitTag(t *testing.T) {
 		})
 	}
 }
+
+// TestSQLObfuscationConfigDeserializationMethod checks if the use of easyjson results in the same deserialization
+// output as encoding/json.
+func TestSQLObfuscationConfigDeserializationMethod(t *testing.T) {
+	cfg, err := json.Marshal(SQLObfuscationConfig{QuantizeSQLTables: true})
+	require.NoError(t, err)
+
+	var in, out SQLObfuscationConfig
+
+	err = json.Unmarshal(cfg, &in)
+	require.NoError(t, err)
+
+	jl := &jlexer.Lexer{Data: cfg}
+	out.UnmarshalEasyJSON(jl)
+	require.NoError(t, jl.Error())
+
```

Comment:
Please remove this line.

Reply:
Updated: https://github.com/DataDog/datadog-agent/pull/8556/commits/2354a436ec591c7cbf376a72af4ef4ee95704c6d